src/tests/NavBar.test.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import {render, screen} from "@testing-library/react";
+import NavBar from "../components/NavBar/NavBar";
+import {BrowserRouter as Router} from "react-router-dom";
+
+describe("NavBar", () => {
+  test("renders NavBar text", () => {
+    render(
+      <Router>
+        <NavBar />
```

Comment:
```suggestion
        <NavBar/>
```
Remove extra spaces in this PR please